distribution/summary/summary.go

```diff
@@ -0,0 +1,128 @@
+package summary
```

Comment:
I suggest moving this package as it is not distribution related, and also used by Artifactory commands.
Perhaps a common utils package is suitable

Reply:
moved into separate package called `summary` will make it to common utils incase there are more common packages

Reply:
This file is also duplicated (still exists in jfrog-cli). Please use the one in core and delete the duplication

Reply:
Yes, This is used in build publish command which is part of artifactory commands when artifactory commands are migrated I will make sure it is removed.